aws-s3/aws-s3.go

```diff
@@ -44,17 +82,15 @@ func (c *AWSS3Client) PutObject(ctx context.Context, content []byte) (string, er
 		Body:   bytes.NewReader(content),
 	}
 
-	res, err := c.client.PutObject(input)
+	_, err := c.client.PutObject(input)
 	if err != nil {
 		return "", err
 	}
 
-	fmt.Printf("response from s3 is: %v \n", res)
 	return newKey, nil
 }
 
 func (c *AWSS3Client) GetObject(ctx context.Context, key string) ([]byte, error) {
-	fmt.Printf("getting object for key: %v\n", key)
 	input := &s3.GetObjectInput{
 		Bucket: aws.String(c.bucket),
 		Key:    aws.String(key),
```

Comment:
In the long run, it would be a good idea to have an s3 bucket for each user. This would make it easier to locate the documents for each user manually. Currently, s3 has a default limit of 100 buckets per account. Hence we're storing all the files in the same bucket `documents`.

Reply:
Additionally, rather than having the get endpoint here, we can simply save the s3 URL with the note purchase agreement. The client can directly call the s3 URL, where they can download the document. This would reduce the maintenance cost to maintain the additional get endpoint.